pages/index.js

```diff
@@ -97,8 +111,14 @@ const HomePage = () => (
                   <p>
                     Run on <code>http://localhost:3000/</code>:
                   </p>
-                  <pre>yarn sketchbook start {"<path/to/design/file.json>"}</pre>
-                  <p>Replace <code>{"<path/to/design/file.json>"}</code> with <code>sketchbook/example.json</code> to run it with an example design file.</p>
+                  <pre>
+                    yarn sketchbook start {"<path/to/design/file.json>"}
+                  </pre>
+                  <p>
+                    Replace <code>{"<path/to/design/file.json>"}</code> with{" "}
+                    <code>sketchbook/example.json</code> to run it with an
+                    example design file.
+                  </p>
```

Comment:
Let's make this `yarn sketchbook start sketchbook/example.json` as I mentioned here:

https://github.com/sketchbook-js/sketchbook/pull/146/files#r656064716